content/showcase/_index.html

```diff
@@ -12,6 +12,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
         <div class="columns is-centered">
             <div class="column is-8">
                 <div class="showcase-gallery columns is-multiline">
+                    {{< showcase-item
+                        src="/img/media/showcase/skribbl.ai.png"
+                        title="Skribbl.AI"
+                        caption="<p>We trained an AI to recognise drawings and give them a score. <a href='https://gothic-welder-360715.ts.r.appspot.com'>Put your doodling skills to the test</a> in our fun web-based game!</p><p><b>2022 Hackathon Best Newcomers</b></p>"
```

Comment:
Just in case your game URL changes or is pulled down, can you remove the direct link to your game from the showcase? Just so this doesn't accidentally become a dead link. You can link it on your Github readme, as you already have done.

Reply:
Fair point 👍